install_pim/docker/installation_docker.rst

```diff
@@ -15,7 +15,7 @@ Getting Akeneo PIM
 ------------------
 
 As our Docker image does not contain Akeneo PIM, you need to download it first.
-This can be done by downloading the archive from our `download page <https://www.akeneo.com/download>`_, or from our partner portal if you have access to the enterprise edition.
+This can be done by downloading the archive from our `download page <https://www.akeneo.com/download>`_, or from `the Portal <https://help.akeneo.com/portal/articles/get-akeneo-pim-enterprise-archive.html?utm_source=akeneo-docs&utm_campaign=installation_archive>`_ if you have access to the enterprise edition.
```

Comment:
remove "our"

Reply:
`downloading the archive from download page`? Seems strange to me. At least replace it by a `the` if you don't want a `our`.